charts/zaakbrug/templates/deployment.yaml

```diff
@@ -154,3 +154,73 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+---
+{{- if .Values.nginx.enabled -}}
```

Comment:
Can we make the condition based on staging.enabled? The API's that are being proxied don't exist/work without staging enabled.